src/Infrastructure/EudonetParis/EudonetParisExecutor.php

```diff
@@ -81,6 +83,9 @@ public function execute(\DateTimeInterface $laterThanUTC): void
                     }
                 }
             }
+
+            $numberOfRegulationsInsideEudonet = $this->eudonetParisExtractor->numberOfRegulations();
+            $numberOfMeasuresInsideEudonet = $this->eudonetParisExtractor->numberOfMeasures();
```

Comment:
(Détail) Utiliser des verbes dans les noms de méthodes

```suggestion
            $numberOfRegulationsInsideEudonet = $this->eudonetParisExtractor->getNumberOfRegulations();
            $numberOfMeasuresInsideEudonet = $this->eudonetParisExtractor->getNumberOfMeasures();
```